src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int ORIGIN = 1;
+    private static final int BOUND = 10;
+    private static final Random random = new Random();
+
+    public static int getRandomInt() {
+        return random.nextInt(ORIGIN, BOUND);
+    }
+
+    public static Figure getRandomFigure() {
+        FigureSupplier figureSupplier = new FigureSupplier();
```

Comment:
1) https://github.com/mate-academy/jv-oop-advanced/blob/master/checklist.md#dont-use-static-methods-in-your-solution
2) figureSupplier is unused and I don't understand why it's created
```suggestion
    public Figure getRandomFigure() {
```